src/main/java/com/onarandombox/MultiverseCore/commands/CloneCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.CommandIssuer;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Conditions;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Single;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.api.MVWorld;
+import org.bukkit.ChatColor;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class CloneCommand extends MultiverseCommand {
+    public CloneCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @Subcommand("clone")
+    @CommandPermission("multiverse.core.clone")
+    @Syntax("<world> <new-world-name>")
+    @CommandCompletion("@mvworlds:type=includeunloaded @empty")
+    @Description("Clones a world.")
+    public void onCloneCommand(CommandIssuer issuer,
+
+                               @NotNull
+                               @Syntax("<world>")
+                               @Description("Current multiverse world.")
```

Comment:
World to clone could be more verbose? I read that as the world I am currently in

Reply:
"World to clone".. yeah might read better.